src/adk/schema/applications/network.json

```diff
@@ -1,22 +1,22 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
     "networks": {
       "type": "array",
-      "items": [
-        {
-          "type": "string"
-        }
-      ]
+      "items": {
+        "type": "string",
+        "readOnly": true,
+        "pattern": "^[-a-zA-Z0-9_]+$"
+      }
     },
     "roles": {
       "type": "array",
-      "items": [
-        {
+      "patternProperties": {
+        "[A-Z][-a-zA-Z0-9_]*": {
```

Comment:
Same comment about roles having to start with a capital letter

Reply:
Changed back